pkg/testcase/pod.go

```diff
@@ -44,8 +46,9 @@ func TestPodStatus(
 				}
 			}
 		}
-	}, "1000s", "5s").Should(Succeed())
+	}, "180s", "30s").Should(Succeed())
```

Comment:
still wrong

Reply:
Why do you think it's wrong? What should be the right approach?

Reply:
a lot of times we have pods that took more than 180s to get ready. ...thats why the value was 1000. !

Reply:
here locally if i dont increase this , always return error.